snowchange/cli.py

```diff
@@ -287,7 +297,7 @@ def create_change_history_table_if_missing(change_history_table, snowflake_sessi
   query = "CREATE TABLE IF NOT EXISTS {0}.{1} (VERSION VARCHAR, DESCRIPTION VARCHAR, SCRIPT VARCHAR, SCRIPT_TYPE VARCHAR, CHECKSUM VARCHAR, EXECUTION_TIME NUMBER, STATUS VARCHAR, INSTALLED_BY VARCHAR, INSTALLED_ON TIMESTAMP_LTZ)".format(change_history_table['schema_name'], change_history_table['table_name'])
   execute_snowflake_query(change_history_table['database_name'], query, snowflake_session_parameters, autocommit, verbose)
 
-def fetch_change_history(change_history_table, snowflake_session_parameters, autocommit, verbose):
+def fetch_change_history(change_history_table, snowflake_session_parameters, autocommit, verbose, dry_run):
```

Comment:
I don't see where dry_run is used in this function?

Reply:
good spot, fixed 👍